src/app/core/services/external-cluster.ts

```diff
@@ -291,8 +292,7 @@ export class ExternalClusterService {
       default:
         return throwError(() => 'Not implemented');
     }
-
-    return this._http.post<ExternalCluster>(url, externalClusterModel, {headers}).pipe(catchError(() => of(null)));
+    return this._http.post<ExternalCluster>(url, externalClusterModel, {headers});
```

Comment:
Where are we catching this error now?

Reply:
observable will be passed to `km-button` and in case of error it handle inside. Due to catching error at this place. Even in case of error from BE it goes under subscribe block which resulted into success and cause redirection to list page without creating. e.g If user unexpectedly enters wrong value.